prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -169,6 +186,8 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
     if (!this.careSettings.length) {
       this.addCareSetting();
     }
+
+    this.healthAuthorities = this.configService.healthAuthorities;
```

Comment:
@neophyte57 your PR has a few errors.  This is one of the issues where the setting of configuration values for health authorities is delayed by not setting them immediately in the constructor.  When you come from the overview to the care setting view the `ngOnInit` lifecycle hook has already been invoked, and your template is being bound with public data, but it's missing the configuration when looping over the health authorities.  You'll want to use the constructor like we do for `careSetting` config and the other components to set up initial component values like configuration.